front/cypress/e2e/project_page_timeline.cy.ts

```diff
@@ -196,6 +196,32 @@ describe('New timeline project', () => {
     cy.get('[data-testid="EventInformation"]').should('exist');
   });
 
+  it('correctly handles phaseNumber URL parameter', () => {
+    const path = `/projects/${projectSlug}`;
+    const pathWithLocale = `/en${path}`;
+
+    cy.location('pathname').should('eq', pathWithLocale);
+
+    // visit first (past phase)
+    cy.visit(`${path}/1`);
```

Comment:
```suggestion
    const pathWithLocale = `/en/projects/${projectSlug}`;

    cy.location('pathname').should('eq', pathWithLocale);

    // visit first (past phase)
    cy.visit(`${pathWithLocale}/1`);
```

Makes the code a bit simpler, we don't need path.